createdisk.sh

```diff
@@ -315,6 +315,10 @@ ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo rpm-ostree install /home/
 # Remove the packages from VM
 ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- rm -fr /home/core/packages
 
+# SCP the kubeconfig file to VM
+${SCP} $1/auth/kubeconfig core@api.${CRC_VM_NAME}.${BASE_DOMAIN}:/home/core/
+${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo mv /home/core/kubeconfig /opt/'
```

Comment:
out of curiosity, why `/opt/` rather than another path? For example `/home/core/` where you put it initially?

Reply:
@cfergeau no reason, I just like to `opt` for option config file :)

Reply:
Hmm this kind of collide with 'normal' use though https://www.tldp.org/LDP/Linux-Filesystem-Hierarchy/html/opt.html